worker/worker.go

```diff
@@ -22,6 +24,7 @@ type Worker interface {
 	ID() string
 	Labels() map[string]string
 	InstructionCache() instructioncache.InstructionCache
+	InjectInstructionCache(instructioncache.InstructionCache)
```

Comment:
Reminder that in `solver-next` this is set on load phase. InstructionCache should become manager concept in the end.